README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-A framework to facilitate the population of XML forms during the usual ingest workflow.
+A framework to facilitate the population of XML forms during the usual ingest workflow. When an object is being an ingested, a page will be added as part of the ingest process that will provide the option to use the populator. Populator options include DOI, EndNote, PMID, and RIS. Populator also works with Islandora XSLT Populator, which can be accessed at /admin/islandora/tools/xslt_populator.
```

Comment:
As the XSLT populator is a submodule of this module, likely shouldn't reference it here in the parent's README.